### ISSUE TITLE
How to set file full path ?

### ISSUE THREAD

**Issue by jsbay** [sidmishraw/topper#15]
Is there a way to set the file full path?
just like:
```vue
<!--
* @Path /user/jsbay/test/src/abc.vue  // this line
* @Created jsbay 2021-04-26 16:51:10
* @Modified jsbay 2021-04-26 18:54:32
* @Description  
-->
```

**Comment by sidmishraw** [comment]
You can make use of the intrinsic parameter called `fileName` in your template to get the full-filesystem path for the file.

Example:
```json

[
    {
        "xml": {
            "headerBegin": "<!--",
            "headerPrefix": "*",
            "headerEnd": "-->",
            "template": [
                "${headerBegin}",
                "${headerPrefix} @Path ${fileName}",
                "${headerPrefix} @Created ${author} ${createdDate}",
                "${headerPrefix} @Modified ${author} ${lastModifiedDate}",
                "${headerPrefix} @Description ${description}",
                "${headerEnd}"
            ]
        }
    }
]

```

**Comment by sidmishraw** [comment]
Nvm, doesn't support that yet, I'll add that in tonight, or tomorrow.

**Comment by jsbay** [comment]
COOL! 
THANKS A LOT!

**Comment by sidmishraw** [comment]
Available with v1.5.0.

Example:

```
{
        "xml": {
            "headerBegin": "<!--",
            "headerPrefix": "*",
            "headerEnd": "-->",
            "template": [
                "${headerBegin}",
                "${headerPrefix} @FileName ${fileName}",
                "${headerPrefix} @PathFromRepositoryRoot ${pathFromRepositoryRoot}",
                "${headerPrefix} @AbsPath ${absFilePath}",
                "${headerPrefix} @RepositoryRootPath ${repositoryRootPath}",
                "${headerPrefix} @Created ${author} ${createdDate}",
                "${headerPrefix} @Modified ${author} ${lastModifiedDate}",
                "${headerPrefix} @Description ${description}",
                "${headerEnd}"
            ]
        }
    }
```

Example output:

```xml
<!--
* @FileName file2.xml
* @PathFromRepositoryRoot dir3/file2.xml
* @AbsPath /user/sid/roughs/dir3/file2.xml
* @RepositoryRootPath /user/sid/roughs
* @Created Sidharth Mishra Wed Apr 28 2021 22:58:69 -0700
* @Modified Sidharth Mishra Thu Apr 29 2021 00:46:46 -0700
* @Description 
-->
```

Dir layout
```
/user/sid/rough/
                         |- dir3
                             |- file2.xml
     
```
